frontend/client/src/real/java/de.amosproj3.ziofa.client/RustClient.kt

```diff
@@ -99,9 +104,15 @@ class RustClient(private val inner: uniffi.client.Client) : Client {
 
     override suspend fun getConfiguration(): Configuration = inner.getConfiguration().into()
 
-    override suspend fun setConfiguration(configuration: Configuration) =
+    override suspend fun setConfiguration(configuration: Configuration): UInt =
         inner.setConfiguration(configuration.into())
 
+    override suspend fun getOdexFiles(pid: UInt): Flow<StringResponse> =
+        inner.getOdexFilesFlow(pid).mapNotNull { it.into() }
+
+    override suspend fun getSymbols(odexFilePath: String): Flow<Symbol> =
```

Comment:
How do i know when i have received all symbols? or is this a continuous stream

Reply:
nevermind, i oversaw the break

Reply:
The flow should just terminate. The rust stream returns None when finished, on the kotlin side we normally break in the flow builder when the flow is cancelled or the stream is empty